src/styles/class-overrides.scss

```diff
@@ -35,6 +35,36 @@
   }
 }
 
+.usa-link {
```

Comment:
Looking good! We can update this to use spacing tokens instead of `$mocto-link-width`, etc. I also started using mixins to save us from repeating code. That might be useful since `.usa-link` and `.usa-link--light` are similar

Reply:
Spacing tokens and mixins used!